save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/FileUploader.kt

```diff
@@ -95,11 +95,6 @@ external interface UploaderProps : PropsWithChildren {
      */
```

Comment:
maybe make this FC more abstract and re-use for sandbox?

Reply:
Probably it is a good idea, but it is not that related to the fixes here. Will add a new issue for that.

Reply:
Not yet, there is just a placeholder for it for now, but your suggestion is very nice, thx